RealmModelGenerator/EntitiesView.swift

```diff
@@ -35,11 +56,39 @@ class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSourc
             cell.title = "Entity"
             return cell
         }
-        cell.title = "\(row)"
+        
+        if let title = self.dataSource?.entitiesView?(self, titleForEntityAtIndex:row) {
+            cell.title = title
+        }
+        
         return cell
     }
     
-    override func prepareForInterfaceBuilder() {
-        super.prepareForInterfaceBuilder()
+    func tableViewSelectionDidChange(notification: NSNotification) {
+        let tableView = notification.object as! NSTableView
+        selectedIndex = tableView.selectedRow
+        self.delegate?.entitiesView?(self, didRemoveEntityAtIndex:selectedIndex!)
```

Comment:
why are you calling the delegate that an entity was removed?
we should call the delegate that the selection has changed